lib/suspenders/actions.rb

```diff
@@ -31,6 +31,28 @@ def expand_json(file, data)
       action ExpandJson.new(destination_root, file, data)
     end
 
+    def gem(*args)
+      options = args.extract_options!
+      name, *versions = args
+
+      parts, message = [quote(name)], name.dup
+
+      if (versions = versions.any? ? versions : options.delete(:version))
+        versions = Array(versions)
+        versions.each do |version|
+          parts << quote(version)
+        end
+        message << " (#{versions.join(", ")})"
```

Comment:
It looks like this variable is not being used. In the [original method](https://github.com/rails/rails/blob/6-1-stable/railties/lib/rails/generators/actions.rb#L21-L47), it is used for logging on this line:

```
log :gemfile, message
```

Do you think logging it would be valuable? Otherwise, we can delete it.

Reply:
Good catch. Updated here: d55520a70085aa71632f708cdebd4269a09c3edd